props/prop_base.h

```diff
@@ -701,6 +701,20 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     FindBlade();
   }
 
+  bool CheckInteractivePreon() {
+    #define USES_INTERACTIVE_PREON(N) \
+    if (current_config->blade##N->current_style() && current_config->blade##N->current_style()->IsHandled(HANDLED_FEATURE_INTERACTIVE_PREON)) return true;
+    return false;
```

Comment:
return false must be after ONCEPERBLADE
Did you test this at all?

Reply:
Fixed, my local version had it reversed when it was in my prop, I had to break up pieces to keep PRs small but it's getting harder to keep track of differences in the files since these have been sitting a while.